wp-content/plugins/core/src/Assets/Admin/JS_Config.php

```diff
@@ -9,8 +9,9 @@ class JS_Config {
 	public function get_data() {
 		if ( ! isset( $this->data ) ) {
 			$this->data = [
-				'images_url'     => trailingslashit( get_stylesheet_directory_uri() ) . 'assets/img/admin/',
-				'block_denylist' => apply_filters( 'tribe/project/blocks/denylist', [] ),
+				'images_url'                 => trailingslashit( get_stylesheet_directory_uri() ) . 'assets/img/admin/',
+				'block_denylist'             => apply_filters( 'tribe/project/blocks/denylist', [] ),
+				'block_theme_service_worker' => defined( 'BLOCK_THEME_SERVICE_WORKER' ) && BLOCK_THEME_SERVICE_WORKER === true,
```

Comment:
Why do we need to support service workers?
I SUGGEST to remove this line, because we don't need it:
```diff
- 'block_theme_service_worker' => defined( 'BLOCK_THEME_SERVICE_WORKER' ) && BLOCK_THEME_SERVICE_WORKER === true,
```